src/app/messages/messages.component.ts

```diff
@@ -2,14 +2,20 @@ import { Component, OnInit } from '@angular/core';
 
 @Component({
   selector: 'app-messages',
-  templateUrl: './messages.component.html',
-  styleUrls: ['./messages.component.css']
+  templateUrl: './messages.component.html'
 })
 export class MessagesComponent implements OnInit {
+  messages: { textContent: string, createdDate: Date }[] = [];
 
   constructor() { }
 
   ngOnInit(): void {
+    // @todo load previous messages
   }
 
+  handleSend(textContent: string) {
+    this.messages = [...this.messages, { textContent, createdDate: new Date() }];
+
+    // @todo send to backend with User ID/Name
```

Comment:
On second thought, I'm being a doofus here. Should send it to backend first as we need to know if it actually sent, but still display it. That way we can allow the user to click retry if it failed.

so need to add message status field.